modules/AOS_PDF_Templates/generatePdf.php

```diff
@@ -362,7 +362,7 @@ function populate_product_lines($text, $lineItems, $element = 'tr')
     return $text;
 }
 
-function populate_service_lines($text, $lineItems, $element = 'tr')
+function populate_service_lines($text, $lineItems, $element = 'tr', $template)
```

Comment:
Hi @jack7anderson7,

We are adding this argument here, but we don't change the place that calls this method.